frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTitles.tsx

```diff
@@ -80,6 +82,7 @@ const SoftwareTitles = ({
         orderDirection,
         orderKey,
         teamId,
+        addedSoftwareToken,
```

Comment:
Nice! I was hoping this was going to be the solution you found :)

A couple small comments for future consideration: 

Are we including this new field in the URL params? If so, can we `omit` it, like we do for scope on line 90 below?

It might be nice to establish a pattern for this approach more generally. Perhaps `addedSoftwareToken` could become something more generic like `refetchAt`.